src/uistate.h

```diff
@@ -56,6 +56,7 @@ class uistatedata : public JsonSerializer, public JsonDeserializer
 
         bool editmap_nsa_viewmode = false;      // true: ignore LOS and lighting
         bool overmap_blinking = true;           // toggles active blinking of overlays.
+        bool overmap_cursor = true;             // toggles overmap cursor
```

Comment:
The value of this variable should persist across saves (see `serialize` and `deserialize` in the same file).